packages/openid4vc-client/package.json

```diff
@@ -25,7 +23,8 @@
   },
   "dependencies": {
     "@aries-framework/core": "0.3.3",
-    "@sphereon/openid4vci-client": "^0.4.0",
+    "@sphereon/oid4vci-client": "file:../../../OID4VCI/packages/client",
+    "@sphereon/oid4vci-common": "file:../../../OID4VCI/packages/common",
```

Comment:
Have you made changes to the openid4vci packages? There's an unstable version that has all the latest changes: [0.4.1-unstable.267](https://www.npmjs.com/package/@sphereon/oid4vci-client/v/0.4.1-unstable.267),

and I think sphereon is going to release a 0.5 version tomorrow-ish

Reply:
Yeah, 0.5 probably gonna be Thursday. The above JSON encoding (and some other v8 fixes) have been resolved last few days